docs/manual/pubsub.md

```diff
@@ -38,8 +38,9 @@ stream of messages where the first element indicates the type of
 message. The commands that are allowed in the context of a subscribed
 client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.
 
-Please note that `redis-cli` will not accept any commands once in
-subscribed mode and can only quit the mode with `Ctrl-C`.
+Please note that when a client using `redis-cli` is in subscription mode, 
+commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used because 
+`redis-cli` will not accept the commands once more and can only quit the mode with `Ctrl-C`.
```

Comment:
Actually redis-cli doesn't accept any commands at all in this mode. Maybe even more clear is:

```suggestion
`redis-cli` will not accept any commands and can only quit the mode with `Ctrl-C`.
```

Reply:
Okay! now it seems clear.